deploy.go

```diff
@@ -312,3 +331,30 @@ func isstale(path string) bool {
 
 	return err != nil
 }
+
+func (p *deploycmd) updateAppConfig(appf *appfile) error {
+	param := clientApps.NewPatchAppsAppParams()
+	param.App = appf.Name
+	param.Body = &models.AppWrapper{
+		App: &models.App{
+			Config: appf.Config,
+		},
+	}
+
+	_, err := p.Apps.PatchAppsApp(param)
+	if err != nil {
+		params := clientApps.NewPostAppsParams() //XXX switch to put when v2.0 Fn
```

Comment:
this works too. clarification of what I meant I guess was that you could switch it back to `deploy->update`, ordering issue aside, even though creating app isn't idempotent in deploy, either. in this case, ordering doesn't matter so much since there are bugs any way we cut it, it's up to you at this point